src/manager/app/controllers/setup.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import os
+
+import requests
+
+ENVIRONMENT = os.getenv("ENVIRONMENT", "dev")
+
+DEFAULT_QDRANT_HOST = (
```

Comment:
we shouldn't assume this is always qdrant - shouldn't the default by chromadb inside of the RAGapp container to keep it simple first?

Reply:
we're using Qdrant right now, but starting with Chroma seems a lot simpler!